### PR TITLE
Updated Profile Model

### PR DESCRIPTION
**Pull request by rajivtitus**
### What this PR does (required):
- Updates to Profile Model
- Included isDogSitter, price, city, profile | cover | gallery photos fields

### Screenshots / Videos (required):
- image

### Any information needed to test this feature (required):
- `open server/models/Profile.js`

### Any issues with the current functionality (optional):
- None

### CODE REVIEW

File: server/models/Profile.js [hunk 1]
```diff
@@ -11,13 +16,18 @@ const profileSchema = new mongoose.Schema({
     },
     phoneNumber: Number,
     address: String,
+    city: String,
     description: String,
     availability: [
         {
             start: Date,
             end: Date,
         }
```
Comment (sina-jamshidi) [inline server/models/Profile.js:25]: I know this isn't part of this ticket but just something I'm thinking about. We might want to discuss availability on Tuesday. Is it going to be a daily availability time? Or is it going to only be certain dates? The way it is set up right now you'd have to say I'm available May 23rd - June 2022 for example, which might not make a lot of sense. The end date should probably be indefinite.

**Review by sina-jamshidi — APPROVED** [review]
These are changes that are going to fit with your profile details UI right? Looks good

**Comment by rajivtitus** [comment]
> These are changes that are going to fit with your profile details UI right? Looks good

Yes, that is correct. Thank you!

File: server/models/Profile.js (hunk 1 above)
Comment (rajivtitus) [reply server/models/Profile.js:25]: That's a very good point. I will have to give it some thought and let's chat about it on Tuesday. Thanks!

**Pull request merged**